### ISSUE TITLE
[4.0] Manual gateway orders appearing in Stripe

### ISSUE THREAD

**Issue by jonleverrier** [craftcms/commerce-stripe#254]
### Description

When our store administrators create a manual order and make a payment against the order using a manual gateway (we're using this to allow customers to pay by 30 days invoice), there is a line in the transaction tab that says "card payment failed", even though we've not interacted with the stripe gateway.

This is also happening on the front-end too when customers choose to pay by manual gateway. 

<img width="1049" alt="Screenshot 2023-10-05 at 10 22 45" src="https://github.com/craftcms/commerce-stripe/assets/2373940/3d9ddade-10f3-4d67-bebf-50ee8f3284e5">

All orders that have been paid for using the manual gateway appear in Stripe as incomplete!

<img width="476" alt="Screenshot 2023-10-05 at 10 30 17" src="https://github.com/craftcms/commerce-stripe/assets/2373940/7f876618-9405-4f6e-afba-5dcd26159a0b">

### Steps to reproduce

1. Setup a manual and stripe gateway
2. Put through a manual order
3. Pay for it using a manual gateway
4. Notice how in the transaction tab, there is a line that says card payment failed 
5. Notice how in the Stripe dashboard the order which was paid for using the manual gateway is in there!

### Additional info

- Craft CMS version:
- Stripe for Craft Commerce version:
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by lukeholder** [comment]
Thanks, this has been fixed for the next release.

To get the fixes early, change your craftcms/commerce and craftcms/commerce-stripe requirements in composer.json to:

```
"require": {
  "craftcms/commerce": "dev-develop#04d8249281d801992f9acc6799db56848c16b4b1 as 4.3.0",
  "craftcms/commerce-
stripe
": "dev-develop#b10503188c4c64efe3f20baf300a669cfbcac16e as 4.0.1",
  "...": "..."
}
```

**Comment by jonleverrier** [comment]
Thanks @lukeholder - that was super quick! If I change my composer.json to get the changes early, do I have to change something back when the new versions of commerce and commerce-stripe come out?

**Comment by jonleverrier** [comment]
@lukeholder I'm getting a `Warning: The lock file is not up to date with the latest changes in composer.json.`

**Comment by lukeholder** [comment]
Sorry, make sure to run `composer update` once updating the composer.json

When the new versions come out, restore the version constraint or use the new version numbers only.

**Comment by jonleverrier** [comment]
Perfect- thank you

**Comment by jonleverrier** [comment]
Hey @lukeholder. Did the fix for this make it into v4.0.1.1? I'm thinking yes, but just double checking

**Comment by lukeholder** [comment]
@jonleverrier yes use the latest version
